server/routes/topics.js

```diff
@@ -1,41 +1,32 @@
 const express = require('express');
-const { path } = require('ramda');
 const Sentry = require('@sentry/node');
 const { logger } = require('../utils/logger');
 
-const fixUrls = element => {
-  const { id, description, href, linkText } = element;
-  switch (element.href) {
-    // TODO: Re-enable when Incentives personalization feature released
-    // case '/content/4204':
-    //   return { id, description, href: '/incentives', linkText };
-    default:
-      return { id, description, href, linkText };
-  }
-};
-
 const createTopicsRouter = ({ topicsService }) => {
   const router = express.Router();
 
   router.get('/', async (req, res, next) => {
     try {
-      const userName = req.user && req.user.getFullName();
-      const establishmentId = path(['session', 'establishmentId'], req);
-      const topics = await topicsService.getTopics(establishmentId);
+      const userName = req.user?.getFullName();
+      const { establishmentName } = req.session;
+
+      if (!establishmentName) {
+        throw new Error('Could not determine establishment!');
+      }
```

Comment:
rather than silently fail decided to hard fail as this should never happen and erroring is more observable